documentation/docs/.vuepress/config.ts

```diff
@@ -75,7 +75,10 @@ export default defineUserConfig({
           {
             text: "ReportSelector",
             children: [
-              "/guide/Frontend/react_components/ReportSelector/ReportSelector.md",
+              "/guide/Frontend/react_components/ReportSelector/ReportSelector",
+              "/guide/Frontend/react_components/ReportSelector/reportSelectorConfig",
```

Comment:
is there a reason why this one is lower case filename, are you doing based on React rules.

Reply:
yes it is a config not a react component

https://react.dev/learn